example/src/DateRangePickerWrapper.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
```

Comment:
Why is this file here?

Reply:
it's similar to the ReactDataGrid file - you need these two folders as a component to have a working calendar. I guess the reason it's like this is b/c I inherited it like this. 

We can talk about it in an hour or so